browser/DEPS

```diff
@@ -1,133 +1,122 @@
 include_rules = [
```

Comment:
git made a mess out of this diff, but it only removes the specific include rules